README.md

```diff
@@ -101,6 +116,10 @@ If configured, the action supports setting the Jira issues's status when the Bug
 - `jira_project_key`
     - string
     - The Jira project identifier
+- `label_field`
+    - string or null
+    - The Jira field to store the Bugzilla status whiteboard labels in, use `null` to disable.
+      Defaults to `labels`
```

Comment:
What different possible Jira fields can the labels be stored in? (Do different Jira projects store labels in different issue elements?)

Reply:
Projects can add custom fields so theoretically any custom field that is added. It has turned out though that we've decided for our project to just not store the labels at all so I could change this setting to a simple boolean to turn it off if that is preferable.

Reply:
I think an optional parameter that can be toggled would be my preference; @grahamalama / @leplatrem what are your thoughts here?

Reply:
I've made that change. If a different project ends up wanting them somewhere else we can address it then.